user/userRoutes.js

```diff
@@ -0,0 +1,113 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../data/helpers/userDb');
+
+const userNameUpper = (req, res, next) => {
+  if (req.body.name) {
+    req.body.name = req.body.name
+      .toLowerCase()
+      .split(' ')
+      .map(s => s.charAt(0).toUpperCase() + s.substring(1))
+      .join(' ');
+  }
+  next();
+};
+
+router.use(express.json());
+
+router.post('/', userNameUpper, (req, res) => {
+  const { name } = req.body;
+  if (!name) {
```

Comment:
Yor if statement is checking for only one thing but your error message is asking for 2 hmm??